doc/man3/OCSP_resp_find_status.pod

```diff
@@ -156,6 +156,10 @@ occurred.
 OCSP_resp_get0_signer() returns 1 if the signing certificate was located,
 or 0 on error.
 
+OCSP_check_validity() returns 1 if B<thisupd> and B<nextupd> are valid time 
+values for an OCSP response given the current time and the constraints
+specified by B<sec> and B<maxsec>, or 0 to indicate an error.
```

Comment:
More accurately:
```
values and the current time + B<sec> is not before B<thisupd> and, 
in case B<maxsec> >= 0, the current time - B<maxsec> is not past B<nextupd>.
Else it returns 0 to indicate an error.
```